snapm/manager/_boot.py

```
@@ -175,15 +75,15 @@ def _get_uts_release():
     return uname()[2]
 
 
-def _get_machine_id():
+def _get_machine_id() -> Optional[str]:
```

Ruff (0.14.2)

Undefined name `Optional`

(F821)